m4/bundle_libdb.m4

```diff
@@ -8,7 +8,7 @@
 
 AC_MSG_CHECKING(Handling bundle_libdb)
 
-db_lib="-L${with_bundle_libdb}/.libs -R${prefix}/lib/dirsrv"
+db_lib="-L${with_bundle_libdb}/.libs -R${prefix}/lib64/dirsrv"
```

Comment:
I wonder if we really need -R option at all. As it is the only place where it is used.

Reply:
Without -R builds fails.